openwpm_utils/blocklist.py

```diff
@@ -1,29 +1,110 @@
-from six.moves.urllib.parse import urlparse
+from urllib.parse import urlparse
+import domain_utils as du
 
-from .domain import get_ps_plus_1
+import abp_blocklist_parser
+import requests
+import pyspark.sql.functions as F
+from pyspark.sql.types import *
 
 
-def get_option_dict(url, top_level_url, content_type):
-    """Build an options dict for BlockListParser
+# Download blocklists
```

Comment:
We don't want these blocklists to be part of the library. Different users will want to parse different lists.